sys/include/isrpipe/read_timeout.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2016 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup isr_pipe ISR Pipe
```

Comment:
This is just copy-pasted from `isrpipe`. You have two options here either remove the `@defgroup` and `@brief` tag and just do `@ingroup isr_pipe` or (since you did not explicitly document that a separate module is needed, I would prefer that)

```C
/**
 * @defgroup isr_pipe_read_timeout  Read timeouts with ISR pipe
 * @ingroup  isr_pipe
 * @brief  ISR -> userspace pipe with timeout
```

Reply:
Ok I do the second one.

Reply:
I aligned the arguments to column 17 to match the other ones.
I also updated the C file to match the isr_pipe_read_timeout.